src/basic_types.hpp

```diff
@@ -91,6 +109,43 @@ TopologicalType GetTopologicalType(TopologicalElement el) {
   }
 }
 
+using TE = TopologicalElement;
+// Returns one if the I coordinate of el is offset from the zone center coordinates,
+// and zero otherwise
+KOKKOS_INLINE_FUNCTION int TopologicalOffsetI(TE el) noexcept {
+  return (el == TE::F1 || el == TE::E2 || el == TE::E3 || el == TE::NN);
+}
+KOKKOS_INLINE_FUNCTION int TopologicalOffsetJ(TE el) noexcept {
+  return (el == TE::F2 || el == TE::E3 || el == TE::E1 || el == TE::NN);
+}
+KOKKOS_INLINE_FUNCTION int TopologicalOffsetK(TE el) noexcept {
+  return (el == TE::F3 || el == TE::E2 || el == TE::E1 || el == TE::NN);
+}
+
+// Returns wether or not topological element containee is a boundary of
```

Comment:
typos in comment

Reply:
I think I just made up a word here, "containee" as in the one who is contained. The internet suggests this is non-standard though...

Reply:
FWIW you can suggest changes for these sorts of things to make the fix one-click